src/contracts/fund/hub/Hub.ts

```diff
@@ -32,6 +42,11 @@ export class Hub extends Contract {
     return super.createDeployment<Hub>(environment, bytecode, from, [args.manager, toHex(args.name)]);
   }
 
+  private async modifierOnlyCreator(from: Address) {
```

Comment:
Let's call this `validateCreator(from: Address)` and add a proper method signature / documentation header.